src/tests/fixtures/comments.fixture.js

```diff
@@ -12,3 +17,25 @@ export const badRequest = {
 export const noTripFound = {
   comment: faker.lorem.sentence()
 };
+
+export const deleteComment = {
+  id: faker.random.number({ min: 10, max: 15 }),
+  userId: loggedInUser.id,
+  subjectId: newTripComment.id,
+  subjectType: 'one-way',
+  comment: faker.lorem.sentence()
+};
+
+export const othersComment = {
```

Comment:
Also here. `othersComment` is not clearly meaningful to someone reading your codes.